test/dummy/.ruby-version

```diff
@@ -0,0 +1 @@
+2.4.1
```

Comment:
Should we not just use 2.5.0?
Thanks for the contribution! This is great work!

Reply:
I think generator crammed it there. Removed it as you should be able to use whatever ruby you want.